### DISCUSSION TITLE
v3.1.0

### DISCUSSION THREAD

**Discussion by danielroe** [nuxt/nuxt#18486] [Announcements]
> **3.1.0** is the first minor release after [Nuxt 3.0](https://github.com/nuxt/framework/releases/tag/v3.0.0) including bug fixes and enhancements.

## 👀 Highlights

- 📖 Lots of improvements to the documentation and website (you can already see them in [nuxt.com](https://nuxt.com))
- 🏝️ Experimental component Islands and [server only components](https://nuxt.com/docs/guide/directory-structure/components#server-components)
- ✨ New `onNuxtReady`, `useNuxtData` and `useSeoMeta` composables
- ﹛ experimental [config schema](https://github.com/nuxt/nuxt/pull/18410) (for module authors)
- 💪 Upgraded to [Nitropack v2](https://github.com/unjs/nitro/releases/tag/v2.0.0)
- ⚡️ Migrated to [Vite 4](https://vitejs.dev/blog/announcing-vite4.html) and [Rollup 3](https://github.com/rollup/rollup/blob/master/CHANGELOG.md#300).

## Changelog

[compare changes](https://github.com/nuxt/nuxt/compare/v3.0.0...v3.1.0)

👉 [Full Changelog](https://github.com/nuxt/nuxt/releases/tag/v3.1.0)

**Comment by gbarrett1988** [comment]


**Reply by danielroe** [reply]
What was the code you had to set that alias?

**Reply by danielroe** [reply]
Yes, that should never have worked. `/<rootDir/` is meant as an example and not meant to be included literally in your aliases.

**Comment by StanleyMasinde** [comment]
This is exciting 🙌🏾. Especially Vite4 and Nitropack 2.
Good job 👏🏾

**Comment by lonewolf-is-coding** [comment]
<!DOCTYPE html>
<html>
<head>
  <title>Car Game</title>
</head>
<body>
  <canvas id="game" width="800" height="600"></canvas>
  <script>
    // Get the canvas element
    var canvas = document.getElementById("game");
    // Get the canvas context
    var ctx = canvas.getContext("2d");

    // Define some variables for the car
    var x = 50;
    var y = 50;
    var speed = 5;
    var carWidth = 50;
    var carHeight = 25;

    // Draw the car on the canvas
    function drawCar() {
      ctx.fillStyle = "red";
      ctx.fillRect(x, y, carWidth, carHeight);
    }

    // Update the car's position
    function updateCar() {
      x += speed;
      if (x > canvas.width) {
        x = -carWidth;
      }
    }

    // Main game loop
    function gameLoop() {
      // Clear the canvas
      ctx.clearRect(0, 0, canvas.width, canvas.height);
      // Update the car
      updateCar();
      // Draw the car
      drawCar();
      // Request another frame
      requestAnimationFrame(gameLoop);
    }

    // Start the game loop
    gameLoop();
  </script>
</body>
</html>

**Reply by lonewolf-is-coding** [reply]
A code for car game guys

**Comment by Celibioux** [comment]
Nuxi CLI act wierd with every command 

![Group 1](https://user-images.githubusercontent.com/33708072/214472376-cb3e8876-ef12-457c-8366-2e0bb1c34af0.png)



is there a way to disable it ? time is not really needed for every command

**Comment by kemalcany** [comment]
@danielroe Good luck at the helm and thanks for the update! 👏🏾
